src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,155 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head = null;
```

Comment:
Don't initialize `head` with `null`; it's redundant since class fields are initialized with default values (null for objects) in Java.